### ISSUE TITLE
Alluxio S3 Proxy requires additional aws sdk settings

### ISSUE THREAD

**Issue by keifgwinn** [3drepo/3drepo.io#1885]
### Description

In order to test using the Alluxio S3 proxy, we need to be able to configure a few extra settings on the AWS SDK.

We need to be able to reconfigure the endpoint to localhost:39999 and the path to the s3 api. 

The details are referenced in this document.
https://docs.alluxio.io/os/user/stable/en/api/S3-API.html

### Goals

- [ ] Able to configure the settings as required

### Tasks

- [ ] Enable AWS SDK in package.json
- [ ] Configuration options able to describe changing the endpoint
- [ ] Configuration options able to describe changing path
- [ ] Configuration options able to describe changing the security (ie. no https)

**Comment by carmenfan** [comment]
3drepo.io[ currently doesn't support write to s3](https://github.com/3drepo/3drepo.io/blob/b058c0cf65d93247eb0ab392b141268f8d607fc4/backend/handler/externalServices.js#L58) (when we wrote this code it was after we ditched S3), which means issues/risks resources upload will not upload to s3. So we need to make sure we add s3 as one of the options in the storeFile function before releasing s3 configurations.

shouldn't be hard to add, we just didn't do it as we had no means to test it.

However bouncer should write to S3 fine. so if you use upload model/view model as your test case it should be fine.... but if you change it and test it with resources first then you won't have to fiddle with the bouncer side and can make sure the web side is all working
